src/dao/DAO.js

```diff
@@ -188,6 +189,45 @@ class DAO<TEntity: { id: EntityID }, TEntityMutator> extends BaseDAO<
     );
   }
 
+  // todo fix flow inconsistensy TResult with entityLoaders - TEntity
+  fetchCustom<TResult>(customQuery: string): LoadObject<TResult> {
+    if (!this._entityLoaderByID.has(customQuery)) {
+      this._entityLoaderByID.set(customQuery, LoadObject.loading());
+      this._emitChanges();
+
+      const daoHeadersArray = (oHandler().oConfig.headers || []: Headers);
+      const daoHeadersObject = daoHeadersArray.reduce(
+        (result: Object, header: Header): Object => ({
+          ...result,
+          [header.name]: header.value,
+        }),
+        {},
+      );
+
+      const baseUrl = ((oHandler().oConfig.endpoint: any): string);
```

Comment:
I'd really prefer that you modify our `o.js` fork to support this pattern. I think it would be best to just add  support for `apply`.

Then you can just use the normal `__resolve` and `__buildHandler`